### ISSUE TITLE
Accessibility is prevented on WSA (Windows Subsytem Android)

### ISSUE THREAD

**Issue by gbraad** [axel358/smartdock#66]
I am unable to change the setting for Accessibility from the SmartDock application.
When checking the settings app specifically I see this shows as a "Restricted setting"

![image](https://github.com/axel358/smartdock/assets/1894/17d82e0b-26d7-41fd-bfd5-ffb92faf9d0e)

What can be done?

**Comment by axel358** [comment]
@gbraad > Settings > Apps > Smartdock > 3 Dot menu (Top right corner) > Allow restricted permissions

**Comment by gbraad** [comment]
I just read https://www.esper.io/blog/android-13-sideloading-restriction-harder-malware-abuse-accessibility-apis about this ;-)

**Comment by gbraad** [comment]
Note: this has to be explicitly enabled from the Settings, as clicking on "Manage" from Smart Dock's wil force the Windows Accessibility settings pane to open.

![image](https://github.com/axel358/smartdock/assets/1894/dd6640ca-0cde-47ad-8a7f-9e63ee8ab6ec)

You want the one of the left, not the one on the right

**Comment by axel358** [comment]
https://github.com/axel358/smartdock#grant-restricted-permissions

Done

**Comment by gbraad** [comment]
Additional note for WSA users. It is not an ideal solution, as unlike Waydroid, WSA does not show the desktop of Android. To make this usable, you have to switch to "Window focus" -> "Independent", else you are unable to use the Dock after the first click you performed.

![Screenshot 2023-07-26 203850](https://github.com/axel358/smartdock/assets/1894/57d6f27f-bf36-4e31-8d52-5e1d9799cac5)

Note: the Windows taskbar and the Smart Dock will conflict if you also not use "Auto hide" (Unfortunately you can not dock the Windows task bar to the top anymore with Win11).
